### PR TITLE
Add CFPs

### PR DESCRIPTION
**Pull request by nimzco**

### CODE REVIEW

File: conferences/2018/javascript.json [hunk 1]
```diff
@@ -651,6 +671,8 @@
     "endDate": "2018-07-20",
     "city": "Edinburgh",
     "country": "U.K.",
+    "cfpUrl": "https://www.papercall.io/scotlandjs-2018",
+    "country": "2018-02-28",
```
Comment (lirantal) [inline conferences/2018/javascript.json]: looks like this is wrong

Comment (nimzco) [reply conferences/2018/javascript.json]: It is but it's been fixed here: https://github.com/tech-conferences/javascript-conferences/pull/181/files

Comment (lirantal) [reply conferences/2018/javascript.json]: oh cool :+1

Comment (nimzco) [reply conferences/2018/javascript.json]: Thanks for the catch though! 😬

**Pull request merged**